### PR TITLE
[Front] - Display users list - admin only

### PR DESCRIPTION
**Pull request by mriiad**
TODO: 

- Fetch users per page using react-query - check src/utils/queries for examples
- Create a new component called `UserCard` to display user's information with style
- Add 3 buttons at the bottom of the `UserCard`: Update, Delete, Mark as donor
- Add pagination in the bottom of the page: check EventsComponent for example
- Add an icon to access /users page via the `MobileNavbar`, the icon is only available for the admin

### CODE REVIEW

File: extranet/src/components/UserCard.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    return (
+
+        <div className="user-card" style={cardStyle}>
+            <div>
+                <p><b>Username :</b> {user.username}</p>
+                <p><b>Email :</b> {user.email}</p>
+                <p><b>Phone Number : </b>{user.phoneNumber}</p>
+                <p><b>Gender :</b> {user.gender}</p>
+                <p><b>IsAdmin :</b> {isAdminText}</p>
```
Comment (mriiad) [inline extranet/src/components/UserCard.tsx:51]: use <Typography></Typography> from MUI instead

File: extranet/src/components/UserCard.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        border: '1px solid white',
+        padding: '30px',
+        width: '370px',
+
+    };
+    const buttons: React.CSSProperties = {
+        display: 'flex',
+        justifyContent: 'space-between',
+        width: '110px',
+    };
```
Comment (mriiad) [inline extranet/src/components/UserCard.tsx:41]: Use makeStyles

File: extranet/generate-react-cli.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "component": {
+    "default": {
+      "path": "src/components",
+      "withStyle": false,
+      "withTest": false,
+      "withStory": false,
+      "withLazy": false
+    }
+  }
+}
```
Comment (mriiad) [inline extranet/generate-react-cli.json:14]: Why adding this file ?

File: extranet/src/components/LoginForm.tsx [hunk 4]
```diff
@@ -34,12 +34,13 @@ const LoginForm = () => {
 		}
 	}, [location]);
 
-	const loginMutation = useMutation((data: FormData) => {
+	const loginMutation = useMutation((data: SignupFormData) => {
+		console.log("################## LOGIN !!")
```
Comment (mriiad) [inline extranet/src/components/LoginForm.tsx:38]:
```suggestion
```

File: extranet/generate-react-cli.json (hunk 3 above)
Comment (Lazrak-chaymae) [reply extranet/generate-react-cli.json:14]: I don't know how this file was added . I only add the user component and user shema in data folder also i add code   in the route folder .

Comment (Lazrak-chaymae) [reply extranet/generate-react-cli.json:14]: it could be added automatically ?

**Comment by mriiad** [comment]
@Lazrak-chaymae please resolve conflicts in this branch with develop

run the following command to fetch the previously committed changes on develop:
`git merge origin/develop`
then resolve conflicts

**Pull request merged**